restorecond: add page (#18458)
tldr-pages/tldr: pages/linux/restorecond.md

# restorecond

> Daemon that monitors file creation and automatically restores SELinux contexts.
> Useful for directories where files are frequently created with incorrect contexts.
> See also: `restorecon`, `semanage-fcontext`.
> More information: <https://manned.org/restorecond>.

> - Start the restorecond daemon:

Reviewer: ```suggestion
- Start the `restorecond` daemon:
```

`sudo restorecond`

> - Run restorecond in verbose mode to see restoration events:

Reviewer: ```suggestion
- Run `restorecond` in [v]erbose mode to see restoration events:
```

`sudo restorecond -v`

> - Run restorecond in debug mode:

Reviewer: ```suggestion
- Run `restorecond` in [d]ebug mode:
```

`sudo restorecond -d`

- Run restorecond in foreground mode (do not daemonize):

> `sudo restorecond -f`

Reviewer: The page you added as in the more info section claims this is for something else. Are you sure this is correct?

Author: You are correct, It was a miss on me. I have rectified and added new commit
